identity-settings-component.html

```diff
@@ -17,7 +17,7 @@
     </div>
 
     <br-input br-model="$ctrl.identityChanges.url"
-      br-options="{icon: 'globe', type: 'url', name: 'url', label: 'Website'}">
+      br-options="{icon: 'globe', type: 'text', name: 'url', label: 'Website'}">
```

Comment:
Need a `<!-- FIXME: change to type 'url'` here and, ideally, an issue to track it.